source/renderer/app/components/wallet/WalletCreateDialog.scss

```diff
@@ -81,9 +73,16 @@
         }
       }
 
+      .tooltip {
+        height: 12px;
+        left: -139px;
+        position: relative;
+        top: 5px;
+        width: 12px;
+      }
```

Comment:
☝️ is this css necessary for the tooltip? it would be great to get rid of any unnecessary css overrides as they are hard to understand

Reply:
Fixed